er_physical.puml

```diff
@@ -24,7 +26,7 @@ entity Player{
     ---
     * Firstname : VARCHAR(50)
     * Lastname : VARCHAR(50)
-    * BirthDate : DATE
+    * Year(BirthDate) : DATE
```

Comment:
YEAR is a function, you can use it in your CHECK constraint, not in the table definition.
This will not work, it's like *DateTime birthDate = year(...)* in C#.